lib/index.ts

```diff
@@ -4,11 +4,36 @@ import * as path from 'path';
 import * as subProcess from './sub-process';
 import { legacyPlugin } from '@snyk/cli-interface';
 import { containsJar, createPomForJar, createPomForJars, isJar } from './jar';
+import * as os from 'os';
 
 export interface MavenOptions extends legacyPlugin.BaseInspectOptions {
   scanAllUnmanaged?: boolean;
 }
 
+export function getCommand(root: string, targetFile: string | undefined) {
+  if (!targetFile) {
```

Comment:
since `targetFile` is `string | undefined`, should I default to `mvn` or should I try to throw an exception here?

Reply:
It's safe to assume the command will be mvn, as the change you are making adds the second command to be run in this plugin :)